plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/commands/SQLCommandAI.java

```diff
@@ -124,9 +124,9 @@ public SQLControlResult handleCommand(@NotNull DBRProgressMonitor monitor, @NotN
             return SQLControlResult.failure();
         }
         AIUtils.disableAutoCommitIfNeeded(
+            new VoidProgressMonitor(),
```

Comment:
You have a monitor available here
```suggestion
            monitor,
```